draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -236,19 +277,19 @@ The document is divided into the following sections:
 
 # Optical Pluggable in a Device with Packet Functions
 
-{{figure-details-packet-optical-device}} shows a packet device from vendor X, which is connected to optical device, equipped with optical pluggables from vendor X and Y. This figure exposes the following internal and external interfaces:
+{{figure-details-packet-optical-device}} shows a host packet device from vendor X, which is connected to optical device, equipped with optical pluggables from vendor X and Y. This figure exposes the following internal and external interfaces:
 
-1. This interface provides the control of packet device packet and optical functions. It provides these functions which can be decoupled such that there is no overlap between the optical and packet control functions.
+1. This interface provides the control of host packet and optical functions. It provides these functions which can be decoupled such that there is no overlap between the optical and packet control functions.
```

Comment:
Yes it is possible.
This can be one deployment model (aka Realization) operator might use. There might be other deployments. The point of this sentence is just separation of responsibilities for IP and Optical